api/src/crocodiles/crocodiles.json

```diff
@@ -1,17 +1,17 @@
 [
-    {
-        "type": "Big croc",
-        "name": "Steven",
-        "image": "http://localhost:3001/images/big.jpeg"
-    },
-    {
-        "type": "Smol croc",
-        "name": "Pam",
-        "image": "http://localhost:3001/images/smol.jpeg"
-    },
-    {
-        "type": "Long croc",
-        "name": "Line",
-        "image": "http://localhost:3001/images/long.jpeg"
-    }
-]
\ No newline at end of file
+  {
+    "type": "Big croc",
+    "name": "Steven",
+    "img": "http://localhost:3001/images/big.jpeg"
```

Comment:
It's ok that we we use localhost for now, since we are in an early stage.
But we need to make this dynamic in order to be able to change the domain when we are going to host it for real.
But maybe we should add a //todo comment about it so we remember it.